src/appkit.cs

```diff
@@ -26692,4 +26729,338 @@ interface NSWindowTabGroup
+
+		[Export ("dimension")]
+		nfloat Dimension { get; }
+	}
+
+	[Mac (10,15)]
+	[BaseType (typeof(NSTouchBarItem))]
+	[DisableDefaultCtor]
+	interface NSPickerTouchBarItem
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[Static]
+		[Export ("pickerTouchBarItemWithIdentifier:labels:selectionMode:target:action:")]
+		NSPickerTouchBarItem Create (NSTouchBarItemIdentifier identifier, string[] labels, NSPickerTouchBarItemSelectionMode selectionMode, [NullAllowed] NSObject target, [NullAllowed] Selector action);
+
+		[Static]
+		[Export ("pickerTouchBarItemWithIdentifier:images:selectionMode:target:action:")]
+		NSPickerTouchBarItem Create (NSTouchBarItemIdentifier identifier, NSImage[] images, NSPickerTouchBarItemSelectionMode selectionMode, [NullAllowed] NSObject target, [NullAllowed] Selector action);
+
+		[Export ("controlRepresentation", ArgumentSemantic.Assign)]
+		NSPickerTouchBarItemControlRepresentation ControlRepresentation { get; set; }
+
+		[Export ("collapsedRepresentationLabel")]
+		string CollapsedRepresentationLabel { get; set; }
+
+		[NullAllowed, Export ("collapsedRepresentationImage", ArgumentSemantic.Strong)]
+		NSImage CollapsedRepresentationImage { get; set; }
+
+		[Export ("selectedIndex")]
+		nint SelectedIndex { get; set; }
+
+		[NullAllowed, Export ("selectionColor", ArgumentSemantic.Copy)]
+		NSColor SelectionColor { get; set; }
+
+		[Export ("selectionMode", ArgumentSemantic.Assign)]
+		NSPickerTouchBarItemSelectionMode SelectionMode { get; set; }
+
+		[Export ("numberOfOptions")]
+		nint NumberOfOptions { get; set; }
+
+		[Export ("setImage:atIndex:")]
+		void SetImage ([NullAllowed] NSImage image, nint index);
+
+		[Export ("imageAtIndex:")]
+		[return: NullAllowed]
+		NSImage ImageAtIndex (nint index);
+
+		[Export ("setLabel:atIndex:")]
+		void SetLabel (string label, nint index);
+
+		[Export ("labelAtIndex:")]
+		[return: NullAllowed]
+		string LabelAtIndex (nint index);
+
+		[NullAllowed, Export ("target", ArgumentSemantic.Weak)]
+		NSObject Target { get; set; }
+
+		[NullAllowed, Export ("action", ArgumentSemantic.Assign)]
+		Selector Action { get; set; }
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Export ("setEnabled:atIndex:")]
+		void SetEnabled (bool enabled, nint index);
+
+		[Export ("isEnabledAtIndex:")]
+		bool IsEnabledAtIndex (nint index);
+
+		// @property (readwrite, copy) NSString * _Null_unspecified customizationLabel;
+		[Export ("customizationLabel")]
+		string CustomizationLabel { get; set; }
+	}
+
+	[Mac (10,15)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSCollectionLayoutSize : NSCopying, INSCopying
+	{
+		[Static]
+		[Export ("sizeWithWidthDimension:heightDimension:")]
+		NSCollectionLayoutSize Create (NSCollectionLayoutDimension width, NSCollectionLayoutDimension height);
+
+		[Export ("widthDimension")]
+		NSCollectionLayoutDimension Width { get; }
```

Comment:
I think `WidthDimension` sounds just as good as `Width`, and it has the advantage of being identical to Swift/Objective-C, so I would go with `WidthDimension` here.

```suggestion
		NSCollectionLayoutDimension WidthDimension { get; }
```